layouts/partials/docs-top-nav.html

```diff
@@ -10,7 +10,7 @@
                     <li role="none"><a role="menuitem" href="/">Pulumi home</a></li>
                     <li role="none"><a role="menuitem" href="/pricing/">Pricing</a></li>
                     <li role="none"><a role="menuitem" href="/blog/">Blog</a></li>
-                    <li role="none"><a role="menuitem" href="/resources/">Events & workshops</a></li>
+                    <li role="none"><a role="menuitem" href="/resources/">Events & worksdhops</a></li>
```

Comment:
```suggestion
                    <li role="none"><a role="menuitem" href="/resources/">Events & worksdhops</a></li>
```

is this *really* a change you want??

Reply:
Ha! It is not. Good catch. 😉